### ISSUE TITLE
Some issues need to be fixed in the Operator Hub section

### ISSUE THREAD

**Issue by jianzhangbjz** [openshift/training#446]
I left [comments](https://github.com/openshift/training/pull/437/commits/033dc636a8b93762983a20ae871f9d929a716f81) in the PR #437, please help fix them. Thanks!

**Comment by thoraxe** [comment]
@jianzhangbjz all comments in that PR were resolved before merge.

If there are still issues can you address them here?

**Comment by jianzhangbjz** [comment]
@thoraxe OK, thanks! I list some issues below about [docs/08-extensions.md](https://github.com/openshift/training/pull/437/commits/033dc636a8b93762983a20ae871f9d929a716f81#diff-63ac239280684c2559718310d68817e0):
1,
> Operators can be installed for all Projects across the cluster or within specific namespaces. Not all Operators support each of these installation methods, but Couchbase does. Choose to install it for all Projects, so all users of the cluster can make Couchbase databases.

Comments:
> me: For beta2, the Couchbase only support the single namespace.
you: fixed already 
me: I'm sorry, but the document wasn't modified. Do you mean we changed the InstallMode of the Couchbase to follow this doc?

Now, this issue still not be addressed.

2,
> Each Operator publisher can create channels for their software, to give adminsitrators more control over the versions that are installed. In this case, Couchbase only has a "preview" channel.

Here, The wrong word `adminsitrators` should be `administrators`.
3, 
> After clicking "Subscribe", the Couchbase entry will show that it is "Installed" within the box.

For now, it should be "Install", not "Subscribe" on the Web console.
4,
> Edit your `cb-example` Couchbase instance to have `4` replicas instead of `3`. A few things will happen:

Here is a little confusing, actually, the `size` field controls the pod number, not the above `replicas`. As below:
```yaml
apiVersion: couchbase.com/v1
kind: CouchbaseCluster
metadata:
  creationTimestamp: '2019-03-07T05:51:45Z'
  generation: 1
  name: cb-example
  namespace: default
  resourceVersion: '175458'
  selfLink: /apis/couchbase.com/v1/namespaces/default/couchbaseclusters/cb-example
  uid: 1735101f-409d-11e9-9c32-0a5a70137f08
spec:
  authSecret: cb-example-auth
  baseImage: registry.connect.redhat.com/couchbase/server
  buckets:
    - conflictResolution: seqno
      enableFlush: true
      evictionPolicy: fullEviction
      ioPriority: high
      memoryQuota: 128
      name: default
      replicas: 1
      type: couchbase
  cluster:
    autoFailoverTimeout: 120
    autoFailoverOnDataDiskIssues: true
    clusterName: cb-example
    indexStorageSetting: memory_optimized
    analyticsServiceMemoryQuota: 1024
    eventingServiceMemoryQuota: 256
    searchServiceMemoryQuota: 256
    autoFailoverMaxCount: 3
    dataServiceMemoryQuota: 256
    autoFailoverOnDataDiskIssuesTimePeriod: 120
    indexServiceMemoryQuota: 256
    autoFailoverServerGroup: false
  exposeAdminConsole: false
  servers:
    - name: all_services
      services:
        - data
        - index
        - query
        - search
        - eventing
        - analytics
      size: 4
  softwareUpdateNotifications: false
  version: 5.5.3-3
```

**Comment by thoraxe** [comment]
@jianzhangbjz it's really hard to follow this without links to code.

> Operators can be enabled for all Projects across the cluster or only within specific namespaces. Not all Operators support each of these installation methods. The Couchbase operator only supports installation for a specific namespace. Make sure mycouchbase is selected to match the project you created earlier.

I think this addresses your 1. It is already in the code.


2 will be fixed in an incoming commit

3 is not accurate. The workflow is:

1. click operator hub
1. click couchbase
1. click install
1. set settings
1. click subscribe

It is accurately documented.

4 doesn't appear in the text.

**Comment by jianzhangbjz** [comment]
Hi, @thoraxe 

> it's really hard to follow this without links to code.

Yes, sorry for this, but I don't know where the code is. I already added comments in [PR 437](https://github.com/openshift/training/pull/437) before.

>In the future, when Couchbase releases a new version to the "preview" channel, the Operator Lifecycle Manager is able to execute a smooth rolling update of the Operator. As an admin, you have control over this process, and can indicate if you want to approve each update, or have it happen automatically. Choose "Automatic" for now.
After clicking "Subscribe", the Couchbase entry will show that it is "Installed" within the box.
3 is not accurate. The workflow is:
...

Yes, the workflow is right. But, no button called `subscribe`.  Should be "After clicking `Approve`, the Couchbase entry will show that it is "Installed" within the box." if we choose the `Manual` approve strategy. If we choose `Automatic`, we don't need to do anything.

> Edit your cb-example Couchbase instance to have 4 replicas instead of 3. A few things will happen:
4 doesn't appear in the text.

This sentence should be "Edit your cb-example Couchbase instance to have 4 `sizes` instead of 3." 
I saw there are many places used the `replicas` word, it's incorrect.  We should replace the `replicas` with `sizes`.

**Comment by thoraxe** [comment]
@jianzhangbjz the code is in this repo. You have to use blame view to see lines.

re: workflow -- the button says "Subscribe" on it. I verified those steps with `4.0.0-0.7`. 

1. click operator hub
1. click couchbase
1. click install
1. set settings
1. click subscribe

re: replicas
I think you are looking at old code/docs. Although I am changing the wording to remove the one reference to replicas. The only place the word _replica_ appears now is in the YAML snippet.

**Comment by jianzhangbjz** [comment]
@thoraxe 
>  the code is in this repo. You have to use blame view to see lines.

Yes, I saw it, [here](https://github.com/openshift/training/blob/master/docs/07-extensions.md), LGTM.

> workflow -- the button says "Subscribe" on it. I verified those steps with 4.0.0-0.7.

Sorry, my misunderstood. It's better to change this sentence to "After clicking "Subscribe", the Couchbase **Operator** entry will show that it is "Installed" within the box."

> I think you are looking at old code/docs. 

Thanks, it looks good to me, now.
